setup.py

```diff
@@ -66,7 +67,16 @@ def read(fname):
     ),
     include_package_data=True,
     package_data={
-        "": ['templates/*', 'static/*./*', '*.js', '*.css', '*.txt'],
+        "mfr.ext.code_pygments": ['static/css/*'],
+        "mfr.ext.ipynb": ['static/*/*', 'templates/*'],
+        "mfr.ext.pdb": ['static/*/*', 'templates/*']
+        "mfr.ext.pdf": ['static/build/*',
+                        'static/web/*.*'
```

Comment:
There are several files of different extensions in `web`, and doing `static/web/*` would attempt to copy the directories therein as well.